VideoLocker/src/main/java/org/edx/mobile/http/Api.java

```diff
@@ -1266,4 +1270,52 @@ public Boolean enrollInACourse(String courseId, boolean email_opt_in) throws Exc
         return false;
     }
 
+
+    /**
+     * Returns course structure object from the given URL.
+     * @param courseId
+     * @param preferCache
+     * @return
+     * @throws Exception
+     */
+    public String getCourseStructure(String courseId, boolean preferCache) throws Exception {
```

Comment:
It looks like we don't call this with ``true`` anywhere. That's coming? Also, kind of seems like the caching policy should be separate from the request generation code itself?